styles.css

```diff
@@ -1 +1,247 @@
-html { margin: 0; }
+* {
+  margin: 0%;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  flex-flow: row wrap;
+  align-items: center;
+}
+
+.fisherman {
+  max-width: 100%;
+  height: 100vh;
+  max-height: 576px;
+  background-image: url('images/Rectangle 1 (1).png');
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  justify-content: center;
+  align-items: center;
+  padding: 0 2em;
+}
+
+h1 {
+  padding: 0 3%;
+  padding-top: 4.8%;
+  font-family: Bebas Neue, Open Sans, sans-serif;
+  font-size: 64px;
+  font-weight: 400;
+  line-height: 77px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+  justify-content: center;
+  margin-top: 0%;
+}
+
+p {
+  padding: 0 4%;
+  padding-top: 12%;
+  font-family: Poppins, Open Sans, sans-serif;
+  font-size: 24px;
+  font-weight: 500;
+  line-height: 36px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+}
+
+.content {
+  max-width: 100%;
+  height: 100vh;
+  max-height: 563px;
+  justify-content: center;
+  flex-direction: column;
+  display: flex;
+  align-items: center;
+  padding: 0 2em;
+}
+
+.fish-hook1 > h2 {
+  font-family: Poppins, Open Sans, sans-serif;
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 27px;
+  letter-spacing: 0;
+  text-align: left;
+  margin: 14.94px 0;
+}
+
+.fish-hook1 > input[type="name"] {
+  background-color: #ededed;
+  border: 1px solid #c7c7c7;
+  width: 427px;
+  height: 65px;
+  top: 845px;
+  left: 41px;
+  margin-bottom: 2em;
+}
+
+.fish-hook2 > h3 {
+  font-family: Poppins, Open Sans, sans-serif;
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 27px;
+  letter-spacing: 0;
+  text-align: left;
+  margin: 14.94px 0;
+}
+
+.fish-hook2 > input[type="email"] {
+  background-color: #ededed;
+  border: 1px solid #c7c7c7;
+  width: 427px;
+  height: 65px;
+  top: 845px;
+  left: 41px;
+  margin-bottom: 2em;
+}
+
+.fish-hook3 > input[type="submit"] {
+  background: #5200ff;
+  width: 427px;
+  height: 65px;
+  top: 845px;
+  left: 41px;
+  font-family: Poppins, Open Sans, sans-serif;
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 27px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #fff;
+  margin-top: 8%;
+  max-height: 563px;
+  justify-content: center;
+  flex-direction: column;
+  display: flex;
+  align-items: center;
+  padding: 0 2em;
+  margin-bottom: 2em;
+}
+
```

Comment:
From here, designs for other screens are included, but this PR says it is mobile only...

I suggest that you keep each screen design in different branches...